.github/workflows/windows.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       run: nmake test VERBOSE_FAILURE=yes TESTS=-test_fuzz* HARNESS_JOBS=4
     - name: install
       # Run on 64 bit only as 32 bit is slow enough already
-      if: $${{ matrix.platform.arch == 'win64' }}
+      if: matrix.platform.arch == 'win64'
```

Comment:
I see from the docs that the bare format is allowed in if: statements, but wouldn't it be more consistent to do what the rest of the expressions in this file do, and use the evaluation syntax, i.e.:
if: ${{ matrix.platform.arch }} == 'win64'

Reply:
@nhorman Adapted!